Limelight/UIAppView.m

```diff
@@ -64,10 +68,15 @@ - (void) updateAppImage {
     if ([_app.id isEqualToString:_app.host.currentGame]) {
         // Only create the app overlay if needed
         _appOverlay = [[UIImageView alloc] initWithImage:[UIImage imageNamed:@"Play"]];
+#if TARGET_OS_TV
+        _appOverlay.adjustsImageWhenAncestorFocused = YES;
+        _appOverlay.userInteractionEnabled = YES;
```

Comment:
The app overlay appears to be getting clipped now on the top and bottom corners of the triangle and also appears to have a strange glow around it which is also clipped when selected.

Reply:
this should be resolved now -- the icon won't animate, but it looks better than a giant overlay icon